run-checks.sh

```diff
@@ -17,5 +17,5 @@
 set -e
 set -o pipefail
 
-flake8 --ignore=W503 --max-complexity=10 brainiak
+flake8 --config setup.cfg brainiak
```

Comment:
We do. It is ignored by default by flake8:
https://github.com/PyCQA/flake8/blob/master/src/flake8/defaults.py#L5